include/mull/JunkDetection/CXX/ASTStorage.h

```diff
@@ -12,6 +12,13 @@ namespace mull {
 
 class MutationPoint;
 
+struct MutationPointSourceInfo {
+  int beginLine;
+  int beginColumn;
+  int endLine;
+  int endColumn;
+};
+
```

Comment:
I don't think it belongs here. I'd suggest putting it into the reporter.
Do you see a different use-case which requires this struct to be defined here?